apps/etterlatte-grunnlag/src/main/kotlin/grunnlag/GrunnlagService.kt

```diff
@@ -111,6 +122,72 @@ class RealGrunnlagService(
         }.associateBy { it.fnr }
     }
 
+    override suspend fun oppdaterGrunnlag(opplysningsbehov: Opplysningsbehov) {
+        val pdlPersondatasGrunnlag = coroutineScope {
+            val persongalleri = opplysningsbehov.persongalleri
+            val sakType = opplysningsbehov.sakType
+            val requesterAvdoed = persongalleri.avdoed.map {
```

Comment:
Disse kunne kanskje vært generalisert noe for å slippe å kjøre nesten samme kode 3-4 ganger.

Reply:
Kom gjerne med forslag :)